routes/users.js

```diff
@@ -69,6 +70,11 @@ router.post("/", async (req, res) => {
       message: "This email has already been registered.",
     });
   } else {
+    const location = reqBody.postalCode ? await LocationUtil.getLocationByPostalCode(reqBody.postalCode) : {
```

Comment:
Put this in because a newly created user would have an `undefined` location and so it wouldn't show properly in the item results page.